content/resources/whitepaper.md

```diff
@@ -4,25 +4,22 @@ toc = "true"
 +++
 
 
-Towards a GitBOM Specification
+GitBOM: Enabling Universal Artifact Traceability In Software Supply Chains
 ===
 
 {{% notification type="info" %}}
-Author: Aeva Black 
-Contact: aeva dot online at gmail / @aeva on OFTC IRC / @aeva.online:matrix.org
+Author: Aeva Black
 
-Updated: 2021-08-26
+Status: DRAFT
 
-Status: WORK-IN-PROGRESS / Seeking Collaborators
+Last updated: 2022-01-25
 {{% /notification %}}
 
 ## Summary
 
-GitBOM is neither `git` nor an SBOM.
+GitBOM is an application of the [git](https://en.wikipedia.org/wiki/Git) [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph), a widely used merkle tree with a flat-file storage format, to the challenge of creating build artifact trees in today's language-heterogeneous open source environments. Contrary to the name's appearance, GitBOM is neither dependent on `git` nor is it a Software Bill Of Materials (SBOM).
```

Comment:
I kind of miss the very punchy opener of "GitBOM is neither 'git' nor a SBOM".

Reply:
I miss it too, but on the other hand, it is generally better to define something in the affirmative.

Reply:
True